### PR TITLE
Some cleanups and fix authorship hooks

### PR DESCRIPTION
**Pull request by kianenigma**

### CODE REVIEW

**Review by joepetrowski — COMMENTED** [review]
Haven't build locally but looks sensible. (We should really get some CI on this repo.)

File: runtime/statemine/src/lib.rs [hunk 1]
```diff
@@ -273,8 +209,7 @@ parameter_types! {
 }
 
 impl pallet_authorship::Config for Runtime {
-	// type FindAuthor = pallet_collator_selection::FindAuthorFromIndex<Self, Aura>;
-	type FindAuthor = ();
+	type FindAuthor = pallet_session::FindAccountFromAuthorIndex<Self, Aura>;
```
Comment (kianenigma) [inline runtime/statemine/src/lib.rs:212]: I mainly wanted to fix this, but then went on the rabbit hole of cleaning.

**Review by JesseAbram — APPROVED** [review]

File: runtime/common/src/lib.rs [hunk 2]
```diff
@@ -15,4 +15,67 @@
...
+	pub const SLOT_DURATION: u64 = MILLISECS_PER_BLOCK;
+
+	// Time is measured by number of blocks.
+	pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
+	pub const HOURS: BlockNumber = MINUTES * 60;
+	pub const DAYS: BlockNumber = HOURS * 24;
+
+	/// We assume that ~10% of the block weight is consumed by `on_initialize` handlers. This is
+	/// used to limit the maximal weight of a single extrinsic.
+	pub const AVERAGE_ON_INITIALIZE_RATIO: Perbill = Perbill::from_percent(10);
```
Comment (joepetrowski) [inline runtime/common/src/lib.rs:74]: Without Collective/Democracy/Staking, is it safe to assume that we could drop this to 5%?

**Review by joepetrowski — APPROVED** [review]

File: runtime/common/src/lib.rs (hunk 2 above)
Comment (kianenigma) [reply runtime/common/src/lib.rs:74]: yeah probably.

Comment (kianenigma) [inline runtime/common/src/lib.rs:74]:
```suggestion
	pub const AVERAGE_ON_INITIALIZE_RATIO: Perbill = Perbill::from_percent(5);
```

File: runtime/common/src/lib.rs [hunk 3]
```diff
@@ -15,4 +15,67 @@
...
+	/// Change this to adjust the block time.
+	pub const MILLISECS_PER_BLOCK: u64 = 6000;
+	pub const SLOT_DURATION: u64 = MILLISECS_PER_BLOCK;
+
+	// Time is measured by number of blocks.
+	pub const MINUTES: BlockNumber = 60_000 / (MILLISECS_PER_BLOCK as BlockNumber);
+	pub const HOURS: BlockNumber = MINUTES * 60;
+	pub const DAYS: BlockNumber = HOURS * 24;
+
+	/// We assume that ~10% of the block weight is consumed by `on_initialize` handlers. This is
```
Comment (kianenigma) [inline runtime/common/src/lib.rs:72]:
```suggestion
	/// We assume that ~5% of the block weight is consumed by `on_initialize` handlers. This is
```

**Pull request merged**